nrfs/src/services/nrfs_usb.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <internal/nrfs_backend.h>
+#include <internal/nrfs_callbacks.h>
+#include <nrfs_usb.h>
+
+typedef struct {
+	nrfs_usb_evt_handler_t handler;
+	bool is_initialized;
+} nrfs_usb_cb_t;
+static nrfs_usb_cb_t m_cb;
```

Comment:
```c
struct nrfs_usb_cb {
	nrfs_usb_evt_handler_t handler;
	bool is_initialized;
};

static struct nrfs_usb_cb m_cb;
```

Reply:
We assumed that typedefs are allowed in the nrfs library.